dpxdt/tools/site_diff.py

```diff
@@ -58,6 +60,14 @@
     'URL prefixes that should not be crawled.')
 
 
+gflags.DEFINE_integer(
```

Comment:
If you've got time it may be worth putting these flags into this file:

https://github.com/bslatkin/dpxdt/blob/master/dpxdt/tools/flags.py

Because then they'd be commonly available and we could put them in other things like url_pair_diff.py.